acp/ideas_module.php

```diff
@@ -128,16 +142,59 @@ public function main($id, $mode)
 			}
```

Comment:
Lets add a check to see if a forum was selected here too.

```php
if (!$cfg_array['ideas_forum_id'])
{
	$errors[] = $this->language->lang('ACP_NO_FORUM_SELECTED');
}
```
Then I think you can decide if we no longer need the check/trigger_error on ideas_forum_id below, if it's not possible to run that code anyways if the forum id has not been set?